templates/taxi/car_detail.html

```diff
@@ -19,4 +13,18 @@ <h1>Drivers</h1>
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+  
+  {% if user in car.drivers.all %}
+    <a href="{% url 'taxi:car-remove-driver' car.id %}" class="btn btn-danger">Delete me from this car</a>
+  {% else %}
+    <a href="{% url 'taxi:car-assign-driver' car.id %}" class="btn btn-primary">Assign me to this car</a>
+  {% endif %}
```

Comment:
Ensure that the logic for assigning and removing drivers is consistent with the permission checks in the views. This will help prevent unauthorized users from modifying driver assignments.